profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.metatag.inc

```diff
@@ -0,0 +1,290 @@
+    ),
+    // Windows & windows mobile tags.
+    'x-ua-compatible' => array(
+      'value' => 'IE=edge',
+    ),
+  );
+  $configs[$config->instance] = $config;
+
+  $config = new stdClass();
+  $config->instance = 'node';
+  $config->api_version = 1;
+  $config->disabled = FALSE;
+  $config->config = array(
+    // tags.
+    'article:modified_time' => array(
+      'value' => '[node:changed:custom:c]',
+    ),
+    'article:published_time' => array(
+      'value' => '[node:created:custom:c]',
+    ),
+    'dcterms.creator' => array(
+      'value' => '[node:author]',
+    ),
+    'dcterms.date' => array(
+      'value' => '[node:created:custom:Y-m-d\\TH:iP]',
+    ),
+    'dcterms.description' => array(
+      'value' => '[node:summary]',
+    ),
+    'dcterms.language' => array(
+      'value' => '[node:language]',
+    ),
+    'dcterms.title' => array(
+      'value' => '[node:title]',
+    ),
+    'description' => array(
+      'value' => '[node:field-meta-description]',
+    ),
+    'keywords' => array(
+      'value' => '[node:field-tags]',
+    ),
+    'og:description' => array(
+      'value' => '[node:field-meta-description]',
+    ),
+    'og:title' => array(
+      'value' => '[node:title] - [site:name] - European Commission',
+    ),
+    'og:type' => array(
+      'value' => '[node:content-type]',
+    ),
+    'og:updated_time' => array(
+      'value' => '[node:changed:custom:c]',
+    ),
+  );
+  $configs[$config->instance] = $config;
+
+  // Before returning these, allow the bundled submodules to override them, thus
+  // extending the "real" defaults before they can then be altered by other
+  // modules.
+  // See hook_metatag_bundled_config_alter() in the API documentation.
+  drupal_alter('metatag_bundled_config', $configs);
+  return $configs;
+}
+
+/**
+ * Implements hook_metatag_info().
+ *
+ * Declaration of tags that are specifically provided by nexteuropa feature.
+ */
+function nexteuropa_metatags_metatag_info() {
+
+  // Create a tag group for nexteuropa tags.
+  $info['groups']['nexteuropa'] = array(
+    'label' => t('Nexteuropa tags'),
+    'description' => t('Add metatag information that respects the IPG guidelines and are required by corporate tools.'),
+    'form' => array(
+      '#weight' => 20,
+    ),
+  );
+
+  // Create a tag group for Europa search tags.
+  $info['groups']['ESS_metatags'] = array(
+    'label' => t('ESS metatags : metatags for Europa Search indexation'),
+    'description' => t('Add metatag information that are necessary for Europa search integration.'),
+  );
+
+  $options = array();
+  if ($vocabulary = taxonomy_vocabulary_machine_name_load('metatags_ess_gf')) {
+    $terms = taxonomy_get_tree($vocabulary->vid);
+    foreach ($terms as $term) {
+      $term_full = taxonomy_term_load($term->tid);
+      $options[$term_full->field_meta_value[LANGUAGE_NONE][0]['value']] = str_repeat('-', $term->depth) . $term->name;
+    }
+  }
```

Comment:
You are closing the condition too soon, we should only create the info tags ess gf entries if there is a taxonomy metatags_ess_gf